keyboards/handwired/split65/stm32/mcuconf.h

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
please run `qmk chibios-confmigrate` on these files and post the small form of these files